docs/upgrade.html

```diff
@@ -15,8 +15,23 @@
  limitations under the License.
 -->
 
+
+
 <h3><a id="upgrade" href="#upgrade">1.5 Upgrading From Previous Versions</a></h3>
 
+<h4><a id="upgrade_10_1" href="#upgrade_10_1">Upgrading from 0.10.0.X to 0.10.1.0</a></h4>
+
+0.10.1.0 is fully compatible with 0.10.0.X. The upgrade can be done one broker at a time by simply bringing it down, updating the code, and restarting it.
+However, please notice the <a href="#upgrade_10_1_breaking">Potential breaking changes in 0.10.1.0</a> before upgrade.
+
+<h5><a id="upgrade_10_1_breaking" href="#upgrade_10_1_breaking">Potential breaking changes in 0.10.1.0</a></h5>
+<ul>
+    <li> The log retention time will be based on the largest timestamp of the messages in a log segment.</li>
+    <li> The log rolling time is now based on the timestamp of the first message in a log segment. i.e. if the timestmp of the first message in the segment is T, the log will be rolled out at T + log.roll.ms </li>
```

Comment:
For the two above, it would probably be useful to say what the previous behaviour was as well.